baricadr/api/__init__.py

```diff
@@ -13,6 +13,13 @@
 api = Blueprint('api', __name__, url_prefix='/')
 
 
+# Endpoint to check if API is running for CLI tests
+# Might return endpoints, version, anything
+@api.route('/', methods=['GET'])
```

Comment:
Ah yes, soing it with /version would be great, the cli could need the info anyway